apps/blockchain/scripts/evm_run.ex

```diff
@@ -0,0 +1,41 @@
+defmodule EVMRun do
```

Comment:
What do you think about `EVM.Runner` or something like that? 

And is there any documentation as to how to use this? It could be just a `moduledoc` here in this file if we don't want to expose this to the world in the README. What do you think?